python/hub/export_yolo_dataset.py

```diff
@@ -0,0 +1,97 @@
+import hub
+import numpy as np
+import os
+import argparse
+
+
+def read_yolo_boxes(fn: str):
+    """
+    Function reads a label.txt YOLO file and returns a numpy array of yolo_boxes
+    for the box geometry and yolo_labels for the corresponding box labels.
+    """
+
+    box_f = open(fn)
```

Comment:
Maybe we could use `with open(fn) as box_f` to have the same pattern as the rest of the file ?